packages/hooks/use-popper-container/index.ts

```diff
@@ -37,10 +35,9 @@ export const usePopperContainer = () => {
     // for this we need to disable the caching since it's not really needed
     if (
       process.env.NODE_ENV === 'test' ||
-      !cachedContainer ||
       !document.body.querySelector(selector.value)
     ) {
-      cachedContainer = createContainer(id.value)
+      createContainer(id.value)
```

Comment:
It's strange, why did we use a variable to save it in the first place?

Reply:
It should be to avoid duplicate creation of DOM. Same as `document.body.querySelector(selector.value)`